src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/RePublishEventToEventStore.cs

```diff
@@ -4,19 +4,24 @@
 using MediatR;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
 
 namespace Esfa.Recruit.Vacancies.Client.Application.EventHandlers
 {
-    public class RePublishEventToEventStore : INotificationHandler<VacancyCreatedEvent>,
-                                                   INotificationHandler<VacancyDraftUpdatedEvent>,
-                                                   INotificationHandler<VacancySubmittedEvent>,
-                                                   INotificationHandler<VacancyReviewApprovedEvent>,
-                                                   INotificationHandler<VacancyReviewReferredEvent>
+    public class RePublishEventToEventStore : 
+                                            INotificationHandler<VacancyCreatedEvent>,
+                                            INotificationHandler<VacancyDraftUpdatedEvent>,
+                                            INotificationHandler<VacancySubmittedEvent>,
+                                            INotificationHandler<VacancyReviewApprovedEvent>,
+                                            INotificationHandler<VacancyReviewReferredEvent>,
+                                            INotificationHandler<UserSignedInEvent>
```

Comment:
In ER-337 (I've just merged) I've re-purposed the UserSignedInEvent so it doesn't need to be republished.  I have added a SetupEmployerEvent and a SetupEmployerEventHandler instead.  Could you remove SetupEmployerEventHandler please and add the handling to this class instead?

Reply:
👍